docs/pages/management/diagnostics/monitoring.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Health Monitoring
+description: Monitoring health and readiness.
+---
+
+## Prerequisites
```

Comment:
Since this section includes instructions to follow while reading this guide, not prerequisites you should have completed before reading it, I think it would make sense to rename it, e.g., to "Enable health monitoring".